C#/Botkit/botbuilder-slack-adapter/SlackMessageTypeMiddleware.cs

```diff
@@ -1,10 +1,38 @@
+/**
+ * Copyright(c) Microsoft Corporation.All rights reserved.
+ * Licensed under the MIT License.
+ */
 using Microsoft.Bot.Builder;
+using Microsoft.Bot.Schema;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
+using System.Text.RegularExpressions;
 using System.Threading.Tasks;
 
+/**
```

Comment:
We should use the `/// <summary>` format in this case, like something like this:
![image](https://user-images.githubusercontent.com/43762887/57935998-61580a80-7899-11e9-8a15-e7a2300c51e6.png)